cmd/kubeadm/app/preflight/checks.go

```diff
@@ -545,6 +545,32 @@ func (sysver SystemVerificationCheck) Check() (warnings, errorList []error) {
 	return warns, nil
 }
 
+// KernelVersionCheck defines struct used for running the kernel version check
+type KernelVersionCheck struct{}
```

Comment:
Sorry, I just noticed that we already have a checker named  `SystemVerificationCheck`, can we use it directly instead of defining the new `KernelVersionCheck`?
Also, KernelVersionCheck seems to have some issues with unit-test https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/129401/pull-kubernetes-unit/1873567703835348992

Reply:
> Also, KernelVersionCheck seems to have some issues with unit-test https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/129401/pull-kubernetes-unit/1873567703835348992

This is fixed with a default reporter.

> Sorry, I just noticed that we already have a checker named SystemVerificationCheck, can we use it directly instead of defining the new KernelVersionCheck?

Yes, the kernel version check is part of `SystemVerificationCheck`. 
SystemVerificationCheck may make the check more thoroughly. Only kernel version check may not be enough as the validator also do some other new checks for cgroup v2.

Reply:
I updated to use SystemVerificationCheck. It seems much clearer. Should we add other checks like `ServiceCheck{Service: "kubelet", CheckIfActive: true}`?

- Do we have cases that the kubelet is stopped before upgradaiton?

Reply:
I think we can keep things the way they are